projects/aca-shared/src/lib/services/app.extension.service.spec.ts

```diff
@@ -228,12 +228,15 @@ describe('AppExtensionService', () => {
         }
       });
 
-      const { files, libraries } = service.documentListPresets;
+      const { libraries } = service.documentListPresets;
+      const files = service.filesDocumentListPreset$;
 
-      expect(files.length).toBe(3);
-      expect(files[0].id).toBe('app.files.thumbnail');
-      expect(files[1].id).toBe('app.files.name');
-      expect(files[2].id).toBe('app.files.securityMarks');
+      files.subscribe((columns) => {
```

Comment:
what in case if that subscribe would not be triggered? Should not we use for example 'done' to ensure it was triggered?

Reply:
added, ty